enginetest/queries/query_plans.go

```diff
@@ -1124,7 +1124,22 @@ Select * from (
 	{
 		Query: `select y, (select 1 from uv where y = 1 and u = x) is_one from xy join uv on x = v order by y;`,
 		ExpectedPlan: "Project\n" +
-			" ├─ columns: [xy.y:1, is_one:4]\n" +
+			" ├─ columns: [xy.y:1, Subquery\n" +
```

Comment:
i'm kind of surprised this doesn't break sometimes. i intentionally separate subquery expressions from their inputs so that correlated subqueries get their inputs. i guess if we stack them deep enough the subqueries still get their inputs. either way this isn't ideal, we need lateral joins to avoid the duplicate exec and naming trade-offs